### PR TITLE
fix(protocol-designer): replace text link style

### PR DESCRIPTION
**Pull request by koji**
# Overview
replace text link style

close RQA-3799

## Review requests
low

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -22739,7 +22739,7 @@ vitest-when@0.5.0:
   dependencies:
     pretty-format "^29.7.0"
 
-vitest@^2.1.8:
+vitest@2.1.8:
```
Comment (jerader) [inline yarn.lock:22742]: is this change needed?

**Review by jerader — APPROVED** [review]
interesting, this aligns with current designs? if so, then lgtm

File: yarn.lock (hunk 1 above)
Comment (koji) [reply yarn.lock:22742]: oh, i think i updated the package.json but didn't push yarn.lock.
the answer is yes.

**Comment by koji** [comment]
> interesting, this aligns with current designs? if so, then lgtm

@jerader 
https://www.figma.com/design/1ot18My22DALIcjdLl5LJh/Helix-Design-System?node-id=4867-4167&m=dev

we need to use blue color for hover and focus-visible.

**Comment by jerader** [comment]
> > interesting, this aligns with current designs? if so, then lgtm
> 
> @jerader https://www.figma.com/design/1ot18My22DALIcjdLl5LJh/Helix-Design-System?node-id=4867-4167&m=dev
> 
> we need to use blue color for hover and focus-visible.

@koji cool thanks! lgtm
